samples/apps/copilot-chat-app/webapi/Skills/PromptSettings.cs

```diff
@@ -34,6 +34,13 @@ public PromptSettings(PromptsConfig promptsConfig)
     /// </summary>
     internal double DocumentContextWeight { get; } = 0.3;
 
+    /// <summary>
+    /// Weight of information returned from planner (i.e., responses from OpenAPI skills).
+    /// Percentage calculated from remaining token limit after memories response and document context have already been allocated.
+    /// Contextual prompt excludes all the system commands.
+    /// </summary>
+    internal double RelatedInformationContextWeight { get; } = 0.75;
```

Comment:
The comments state that this is calculated from the remaining token limit, but in practice this is being hard coded, can you explain why in the comments

Reply:
Good catch! I forgot to change that during testing, ChatSkill leverages the PromptSettings variable now